src/backends/gr.jl

```diff
@@ -268,16 +268,16 @@ end
 
 
 # using the axis extrema and limit overrides, return the min/max value for this axis
-gr_x_axislims(sp::Subplot) = axis_limits(sp.attr[:xaxis], true)
-gr_y_axislims(sp::Subplot) = axis_limits(sp.attr[:yaxis], true)
-gr_z_axislims(sp::Subplot) = axis_limits(sp.attr[:zaxis], true)
+gr_x_axislims(sp::Subplot) = axis_limits(sp.attr[:xaxis], false)
+gr_y_axislims(sp::Subplot) = axis_limits(sp.attr[:yaxis], false)
+gr_z_axislims(sp::Subplot) = axis_limits(sp.attr[:zaxis], false)
 gr_xy_axislims(sp::Subplot) = gr_x_axislims(sp)..., gr_y_axislims(sp)...
 
 function gr_lims(axis::Axis, adjust::Bool, expand = nothing)
     if expand != nothing
         expand_extrema!(axis, expand)
     end
-    lims = axis_limits(axis, true)
+    lims = axis_limits(axis, false)
```

Comment:
The bug was that the endpoints were not being included... I'll look for the
issue later.

On Wednesday, June 1, 2016, Josef Heinen <notifications@github.com> wrote:

> In src/backends/gr.jl
> <https://github.com/tbreloff/Plots.jl/pull/294#discussion_r65387283>:
>
> >  gr_xy_axislims(sp::Subplot) = gr_x_axislims(sp)..., gr_y_axislims(sp)...
> >
> >  function gr_lims(axis::Axis, adjust::Bool, expand = nothing)
> >      if expand != nothing
> >          expand_extrema!(axis, expand)
> >      end
> > -    lims = axis_limits(axis, true)
> > +    lims = axis_limits(axis, false)
>
> I'd prefer to fix the marker problem in the right place. Do you have an
> example?
>
> Right now, markers are drawn if they are in the clip rectangle (closed
> intervals for X, Y, *including* the endpoints): xmin <= x <= xmax && ymin
> <= y <= ymax
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/tbreloff/Plots.jl/pull/294/files/a088b00f708037183650a54a56a36a185550532e#r65387283>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA492vgpoZNxZubCwp_3a5oemm8BzBKbks5qHakYgaJpZM4IrqeN>
> .
>